src/main/java/de/tum/in/www1/artemis/service/LectureImportService.java

```diff
@@ -172,7 +173,7 @@ else if (importedLectureUnit instanceof ExerciseUnit) {
      * @param importedAttachment The original attachment to be copied
```

Comment:
The `@param` annotations in the comment can be extended to align with the current method signature.